week-1/Homework/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,4 @@
-// Start by creating a variable `message`
+var name = "Burçak";
+var nameLength = name.length;
```

Comment:
Have you learned the difference between `const`, `let` and `var`? As a rule of thumb, you can use always const by default which means you're storing a constant. If you need to change the value of the variable, then you can use `let`. `var` will be visible in the global scope and it's very uncommon nowadays. This little change lets you save some memory.